docker-compose.yml

```diff
@@ -0,0 +1,45 @@
+version: "3.8"
+services:
+  db:
+    image: postgres:14.3-alpine
+    volumes:
+      - ./data/db:/var/lib/postgresql/data/
+    environment:
+      - POSTGRES_USER=postgres
+      - POSTGRES_PASSWORD=postgres
+      - POSTGRES_DB=postgres
```

Comment:
DB name could be more descriptive (ex: jsuite, or whatever we plan on renaming the whole project to)

Reply:
I'll leave it at that while we decide what we're renaming the project, too but i'll open an issue for it